pkg/querier/stats/stats.proto

```diff
@@ -30,4 +30,6 @@ message Stats {
   uint32 split_queries = 6;
   // The number of index bytes fetched on the store-gateway for the query
   uint64 fetched_index_bytes = 7;
+  // The estimated number of series to be fetched for the query
+  uint64 fetched_series_estimate = 8;
```

Comment:
[nit] I suggest to call it estimated series cause it's just number of... estimated series.

```suggestion
  uint64 estimated_series = 8;
```

Reply:
I chose this naming because the actual value is contained in `fetched_series_count` and wanted to show the relationship through similar naming. Maybe to keep the similarity somewhat we could use `estimated_series_count`?

Reply:
> estimated_series_count

LGTM